chapters/refereecommands.adoc

```diff
@@ -148,11 +148,19 @@ Penalty Kicks are used to punish teams that received multiple <<Yellow Card, yel
 
 ==== Yellow Card
 .Definition
-A yellow card can only be given during <<Halt, halt>>.
 
 If the yellow card is shown as a result of <<Unsporting Behavior, unsporting behavior>>, the referee may decide to immediately <<Halt, halt>> the match. In this case, the match continues with a direct free kick for the other team.
 
-Upon receipt of a yellow card, the number of robots allowed on the field for the penalized team decreases by one. If, after this decrease, the team has more robots than permitted on the field, a robot must be <<Robot Substitution, taken out>> before <<Resuming The Game, play resumes>>. The penalized team can choose the robot to remove.
+Upon receipt of a yellow card, the number of robots allowed on the field for the penalized team decreases by one. If, after this decrease, the team has more robots than permitted on the field, a robot must be <<Robot Substitution, taken out>>.
+
+After the yellow card is issued the game will not be stopped, the team will have 10 seconds to <<Robot Substitution, automatically remove the robot>>. If a robot is not taken out within time, the game is stopped for <<Robot Substitution, manual substitution>>.
+
+NOTE: There are some fouls that causes game stoppage, e.g. dropping parts. Therefore, if any of these fouls happen within 10 seconds of the yellow card, teams are allowed to replace the robot.
```

Comment:
that's not, what I meant :D

A yellow card can be given for different reasons, some of which do not cause the game to be stopped, e.g. multiple fouls, where the last foul was a robot crash, which does not stop the game. But if the last foul that caused the yellow card (possibly indirectly through multiple fouls) is for example robot tipping over, than the game is obviously be stopped (actually even before the yellow card is issues).

Reply:
Got it